content/en/docs/collector/deployment/multiple-collectors.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Multiple Collectors
+description: Considerations for single-writer responsibility when deploying multiple collectors in a gateway configuration.
+weight: 3
+---
+
+## Deploying Multiple Collectors
+
+When deploying multiple collectors in a gateway configuration, it's important to
+ensure that all metric data streams have a single writer and a globally unique identity.
+
+
+### The Single-Writer Principle
+
+The Single-Writer Principle refers to employing a single logical writer for a particular resource.
+Concurrent access from multiple applications that modify or report on the same data can lead to data
+loss or, at least, degraded data quality. In gateway collector deployments, applying this principle guards
+against sending inconsistent data to the backend.
+
+### Further Context
+
+All metric data streams within OTLP must have a [single writer](https://opentelemetry.io/docs/specs/otel/metrics/data-model/#single-writer).
+In a system with multiple collectors, the single-writer principle is most relevant for receivers that generate their 
+own metrics, such as scraping receivers or the host metrics receiver. 
+This is most important for receivers that create their own metrics, such as pull based scraping 
+that targets a specific metric source.
```

Comment:
I believe the two last statements are saying the same things. Perhaps complement the first with extra info from the second?

```suggestion
In a system with multiple collectors, the single-writer principle is most relevant for receivers that generate their
own metrics or target a specific metric source, such as scraping receivers, host metrics receiver, kubelet stats receiver, and similar.
```